instrumentation/jmx-metrics/library/src/main/java/io/opentelemetry/instrumentation/jmx/yaml/MetricStructure.java

```diff
@@ -136,36 +136,84 @@ private List<MetricAttribute> addMetricAttributes(Map<String, Object> metricAttr
     return list;
   }
 
-  private static MetricAttribute buildMetricAttribute(String key, String target) {
+  // package protected for testing
+  static MetricAttribute buildMetricAttribute(String key, String target) {
+    String errorMsg =
+        String.format("Invalid metric attribute expression for '%s' : '%s'", key, target);
+
+    String targetExpr = target;
+
+    // an optional modifier may wrap the target
+    // - lowercase(param(STRING))
+    boolean lowercase = false;
+    String lowerCaseExpr = tryParseFunction("lowercase", targetExpr, target);
```

Comment:
I guess for now it doesn't matter, but if you plan to add more transforms then it would be beneficial to restructure the parsing logic so that `lowercase(lowercase(const(Hello)))` would be valid.